### PR TITLE
Refactoring code chunk

### PR DESCRIPTION
**Pull request by RuffaloLavoisier**
with #399

### CODE REVIEW

File: src/apps/tools/OswAppDistStats.cpp [hunk 1]
```diff
@@ -9,6 +9,8 @@
 #include <osw_hal.h>
 #include <osw_pins.h>
 
+#define WEEK 7
+
```
Comment (simonmicro) [inline src/apps/tools/OswAppDistStats.cpp:13]: I do not think define-ing the number of week days is really that useful - especially as this define is never reuzsed and just re-defined in each source file. If you really want to label it (and I stretch: I you _really_ want to), maybe declare a `constexpr` int before the `for` call and avoid using the pre-processor for this.

File: src/apps/watchfaces/OswAppWatchface.cpp [hunk 2]
```diff
@@ -25,33 +27,34 @@ const char* OswAppWatchface::getAppName() {
 }
 
 #ifdef OSW_FEATURE_STATS_STEPS
-void OswAppWatchface::drawStepHistory(OswUI* ui, uint8_t x, uint8_t y, uint8_t w, uint8_t h, uint32_t max) {
+void OswAppWatchface::drawStepHistory(OswUI* ui, uint8_t posX, uint8_t posY, uint8_t width, uint8_t height, uint32_t max) {
     OswHal* hal = OswHal::getInstance();
     OswUI::getInstance()->resetTextColors();
     uint32_t weekDay = 0;
     uint32_t dayOfMonth = 0;
+    uint8_t index = 0;
```
Comment (simonmicro) [inline src/apps/watchfaces/OswAppWatchface.cpp:35]: Why did you move this out of the `for`-loop if you are not accessing it outside the loop? Just leave it in there to reduce the potential of variable-name collisions.

File: src/apps/watchfaces/OswAppWatchface.cpp [hunk 3]
```diff
@@ -25,33 +27,34 @@ const char* OswAppWatchface::getAppName() {
...
         hal->gfx()->setTextCenterAligned();  // horiz.
         hal->gfx()->setTextBottomAligned();
         hal->gfx()->setTextSize(1);
-        hal->gfx()->setTextCursor(CENTER_X, y - 1);
+        hal->gfx()->setTextCursor(CENTER_X, posY - 1);
 
         hal->gfx()->print(hal->environment()->getStepsToday() + (OswConfigAllKeys::settingDisplayStepsGoal.get() ? String("/") + max:""));
 
-        hal->gfx()->setTextCursor(CENTER_X, y + 1 + 8 + w * 4);
+        hal->gfx()->setTextCursor(CENTER_X, 221);
```
Comment (simonmicro) [inline src/apps/watchfaces/OswAppWatchface.cpp:57]: I would not hard-code such value! The display size is dynamic and the placement of the boxes is too. If you want you could try to tell the compiler to pre-calculate it using a `constexpr`, but this math is really not a slowdown.

File: src/apps/watchfaces/OswAppWatchfaceMix.cpp [hunk 4]
```diff
@@ -24,8 +24,8 @@ const char* OswAppWatchfaceMix::getAppName() {
 void OswAppWatchfaceMix::analogWatchDisplay() {
     OswHal* hal = OswHal::getInstance();
     uint32_t second = 0;
-    uint32_t minute = 0;  // Unused, but required by function signature
-    uint32_t hour = 0;    // Unused, but required by function signature
+    uint32_t minute = 0;
+    uint32_t hour = 0;
 
     hal->getLocalTime(&hour, &minute, &second);
```
Comment (simonmicro) [inline src/apps/watchfaces/OswAppWatchfaceMix.cpp:30]: Removed it before, but here... Not?

**Review by simonmicro — CHANGES_REQUESTED** [review]
Some good changes, some... Why?

**Comment by RuffaloLavoisier** [comment]
LGTM.

**Comment by RuffaloLavoisier** [comment]
to #405.

**Pull request closed without merging**